crouton.php

```diff
@@ -306,11 +306,26 @@ public function tabbedUi($atts, $content = null)
                 "exclude_zip_codes" => null,
                 "show_distance" => '0',
                 "custom_query" => null,
-                "used_formats" => '0'
+                "used_formats" => '0',
+                "show_map" => '0',
+                "max_zoom_level" => 15
             ), $atts));
             if ($show_distance == '1') {
                 wp_enqueue_script("bmlt-tabs-distance", plugin_dir_url(__FILE__) . "js/bmlt_tabs_distance.js", array('jquery'), filemtime(plugin_dir_path(__FILE__) . "js/bmlt_tabs_distance.js"), true);
             }
+            if ($show_map == '1' && $this->options['google_api_key'] != '') {
+                wp_enqueue_script("markerclusterer", plugin_dir_url(__FILE__) . "js/markerclusterer.js", array('jquery'), filemtime(plugin_dir_path(__FILE__) . "js/markerclusterer.js"), true);
+                wp_enqueue_script("oms", plugin_dir_url(__FILE__) . "js/oms.min.js", array('jquery'), filemtime(plugin_dir_path(__FILE__) . "js/oms.min.js"), true);
+                wp_enqueue_script('google-maps', 'https://maps.googleapis.com/maps/api/js?key=' . $this->options['google_api_key'], '', '');
+                wp_enqueue_script("bmlt-tabs-map", plugin_dir_url(__FILE__) . "js/bmlt_tabs_map.js", array('jquery'), filemtime(plugin_dir_path(__FILE__) . "js/bmlt_tabs_map.js"), true);
+                wp_localize_script('bmlt-tabs-map', 'bmltTabsMap', array(
+                    'pluginUrl' => plugin_dir_url(__FILE__),
+                    'maxZoomLevel' => $max_zoom_level,
+                ));
+            }
+            if ($show_map == '1' && $this->options['google_api_key'] == '') {
```

Comment:
these two if statements can be combined in a sense.

Reply:
not sure if thats what you meant but in a sense i combined them ha